### PR TITLE
Update the arts_thoery document

### PR DESCRIPTION
**Pull request by gongxunhlj**
fix some errors.

### CODE REVIEW

**Comment by gongxunhlj** [comment]
Update the arts_thoery document

File: doc/uguide/absorption_theory.tex [hunk 1]
```diff
@@ -2527,7 +2527,7 @@ \subsubsection{MPM93 water vapor absorption model}
 coefficients:
 \begin{equation}
   \label{eq:abs_theory:mpm93:absline}
-  \alphampmnl = 0.1820 \cdot \nu \cdot \phzo \cdot 
+  \alphampmnl = 0.1820 \cdot \nu \cdot
```
Comment (olemke) [inline doc/uguide/absorption_theory.tex]: First of all thanks a lot for your continued work on corrections for the theory guide. We really appreciate that (and are surprised how many typos there actually are ;-) ).
I have one question about the change in this line: Why did you remove the PH2O in this place? It seems to me it should be present here just like in all other MPM models?

**Comment by gongxunhlj** [comment]
This software is excellent, and I try to learn it.
When I read the arts theory, I code a python program. But the result was different from that in ref. [1]. So I compare MPM93 in arts theory with ref. [1]:
in ref [1], the pressures are only present in line intensity equations, not in alpha, so is MPM93 oxygen model in arts theory. But in MPM93 water vapor model of arts theory, the PH2O is present in both alpha and line intensity. I think one of them should be removed.
I didn't check the MPM89 and MPM87 water vapor models. I think maybe there are the same problems in them.
Please check them (include MPM93). Maybe I'm wrong.


Reference:
[1] Liebe, H. J., G. A. Hufford, and M. G. Cotton, Propagation modeling of moist air and suspended water/ice particles at frequencies below 1000 GHz., in AGARD 52nd Specialists Meeting of the Electromagnetic Wave Propagation Panel, Palma de Mallorca, Spain, 1993


 
From: Oliver Lemke
Date: 2020-02-25 14:45
To: atmtools/arts
CC: gongxunhlj; Author
Subject: Re: [atmtools/arts] Update the arts_thoery document (#133)
@olemke commented on this pull request.


In doc/uguide/absorption_theory.tex:
> @@ -2527,7 +2527,7 @@ \subsubsection{MPM93 water vapor absorption model}
 coefficients:
 \begin{equation}
   \label{eq:abs_theory:mpm93:absline}
-  \alphampmnl = 0.1820 \cdot \nu \cdot \phzo \cdot 
+  \alphampmnl = 0.1820 \cdot \nu \cdot

First of all thanks a lot for your continued work on corrections for the theory guide. We really appreciate that (and are surprised how many typos there actually are ;-) ).
I have one question about the change in this line: Why did you remove the PH2O in this place? It seems to me it should be present here just like in all other MPM models?
